resources/system-overview.md

```diff
@@ -0,0 +1,105 @@
+# The Shape of Eligibility Systems
+
+It can be helpful to undertand the general shape of the systems that you will be intereacting with. There is much variation in capability, acrhtiecture, and even nomenclature across states and vendors. This document tried to capture some generalities.
+
+## System Overview
+At the widest perspective the Medicaid eligibility system includes a number of interconnected components: a datastore, a system implementing the business logic of the program, often a separate “rules engine” that implements the business logic specific to eligibility determinations, connections to a variety of state and federal data sources, a system for generating notifications across a range of media, a number of specialized user interfaces for different audiences, and a mechanism for generating reports and dashboards.
+
+These systems process and store both PII and PHI, which adds additional constraints to their design.
+
+![block diagram of a medicaid elgibility system including web portals, buisness logic, rules engines, and data sources](/images/system_diagram.svg)
+
+### User Portal
+Narrowly: a set of web pages where the beneficiary can: learn about the program, apply for benefits, check progress of their case, respond to requests for information, provide information about change of circumstances events, learn about successful renewals, or complete a manual renewal.
+
+More broadly, the (potential) beneficiary can interact with the system in many other ways including US mail, text messages, phone-based customer support, in-person customer support from state or county staff, and in-person support from officially designated navigators.
+
+Responses to request for information might involve filling out an electronic form, or uploading photos or scans of a document via the user portal, or submitting physical documents via drop box, US Mail, or even fax.
```

Comment:
This would be a great spot to describe in a commonly-seen issue: upload file size limits!

Reply:
done!